README.md

```diff
@@ -1,18 +1,75 @@
-# QleetOS User Documentation
+# qleetctl
```

Comment:
This repo is for user documentation - not qleetctl.  Not sure why we want to change the title here